.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+exclude: 'src/pip/_vendor/'
+
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+    # - id: check-builtin-literals  # TODO: enable when fixed.
+    - id: check-added-large-files
```

Comment:
I would dedent these 2 spaces.

Reply:
I'd say that's a formatting/style thing -- is it OK if we address this in a follow up?

Reply:
Not making a style argument, just looking for consistency in this file. Compare [here](https://github.com/pypa/pip/pull/7078/files/096cfb2de093a4d8331f8473aaf11ce51a6b3ed8#diff-82c3b36709926de3a1b27bea0d8258ccR3-R5)

```
repos:
- repo: https://github.com/pre-commit/pre-commit-hooks
  rev: v2.3.0
```

with [here](https://github.com/pypa/pip/pull/7078/files/096cfb2de093a4d8331f8473aaf11ce51a6b3ed8#diff-82c3b36709926de3a1b27bea0d8258ccR29-R31)

```
  hooks:
    - id: mypy
      exclude: docs|tests
```

While I personally prefer the first format over the second, I'm OK with either as long as it is consistent throughout.

That said if there's some reason not to make the change I won't block this PR over it. :) Everything else looks good to me.

Reply:
Fair enough.